### PR TITLE
Admin endpoint for merging users

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala [hunk 1]
```diff
@@ -56,8 +56,33 @@ class AdminUserController @Inject() (
     keepToCollectionRepo: KeepToCollectionRepo,
     userIndex: UserIndex,
     userNotifier: UserNotifier,
+    emailAddressRepo: EmailAddressRepo,
+    invitationRepo: InvitationRepo,
     clock: Clock) extends AdminController(actionAuthenticator) {
 
+  def merge(fromUserId: Id[User], toUserId: Id[User]) = AdminHtmlAction { implicit request =>
+    // This doesn't do a complete merge. It's designed for cases where someone accidentally creates a new user when
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala:64]: Nice :)

Comment (gmethvin) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala:64]: Eishay had an issue with his user today and I decided it was finally time to make an easy way to do this without having to resort to database/cache hackery.

Comment (eishay) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala:64]: cool!

File: kifi-backend/modules/shoebox/conf/shoebox.routes [hunk 2]
```diff
@@ -189,6 +189,7 @@ GET     /admin/commentIndex/reindex @com.keepit.controllers.admin.AdminCommentIn
 GET     /admin/commentIndex/dumpDoc/:commentId  @com.keepit.controllers.admin.AdminCommentIndexController.dumpLuceneDocument(commentId: Id[Comment])
 
 GET     /admin/users                @com.keepit.controllers.admin.AdminUserController.allUsersView
+GET     /admin/users/merge          @com.keepit.controllers.admin.AdminUserController.merge(from: Id[User], to: Id[User])
```
Comment (2is10) [inline kifi-backend/modules/shoebox/conf/shoebox.routes:192]: `POST`

**Pull request merged**